examples/m3.kl

```diff
@@ -0,0 +1,6 @@
+#lang kernel
+
+(import (shift "m2.kl" 1))
+(import "m2.kl")
```

Comment:
Only the non-shifted import is used, but the shifted import is needed in order to trigger the bug. Normally, importing the same module with and without a shift does not cause any problem, but in this case it does.